src/Adapter/JsPull.php

```diff
@@ -51,7 +51,7 @@ public function notify($current, $max, $percent, $timeTaken, $timeRemaining, $te
             'finished'      => false
         ];
 
-        $data = Json::encode($arguments);
+        $data = json_encode($arguments);
```

Comment:
Important: `Json::encode()` and `Json::decode()` caused **exceptions** to be thrown, when using `laminas/laminas-json`.

We want to use `JSON_THROW_ON_ERROR` here.

Reply:
`json_*en*code` should be infallible here, but I added the flag for good measure, as requested.

I guess that's why defaults matter :innocent:

Reply:
cheeky :P